Doc/library/bdb.rst

```diff
@@ -228,9 +276,9 @@ The :mod:`bdb` module also defines two classes:
 
       Stop when returning from the given frame.
 
-   .. method:: set_until(frame)
+   .. method:: set_until(frame[, lineno=None])
```

Comment:
```suggestion
   .. method:: set_until(frame, lineno=None)
```
The brackets are unnecessary, the default already indicates the param is optional.